### ISSUE TITLE
Update Contribution guide with Team Member pull request policy

### ISSUE THREAD

**Issue by tcr** [tessel/project#52]
We specced out a PR policy that ensures pull requests are attributed to their original author, but are augmented with any appropriate changes or fixes that should be done before they are merged into master.

This ensures attributing credit where it is due, but also giving examples of improving code quality in the future.

**Comment by Frijol** [comment]
I thought this was meant to be notes on the Team Member guide? To help TMs (ha) figure out how to review submissions from first-time contributors.
